app/constants/workload-report-status.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  PENDING: 'PENDING',
+  INPROGRESS: 'IN-PROGRESS',
+  COMPLETEDWITHFAILURES: 'COMPLETED-WITH-FAILURES',
```

Comment:
Are we going with this naming convention for constants? I may have used SOME_KINDA_EXAMPLE in some places for the reference name. I can update where found if so.